hack/kubevirtci.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+KUBEVIRTCI_TAG=${KUBEVIRTCI_TAG:-$(curl -sfL https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)}
+export KUBEVIRTCI_TAG
+
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.21}
```

Comment:
k8s-1.21 is going EOL on 2022-06-28, can you please use newer?

Reply:
I changed it to 1.23